.github/workflows/comment-build-test.yml

```
@@ -0,0 +1,64 @@
+
+jobs:
+  parse-and-trigger:
+    if: github.event.issue.pull_request && startsWith(github.event.comment.body, '/run-ci')
+    runs-on: ubuntu-latest
+    outputs:
+      runner: ${{ steps.parse.outputs.runner }}
+      should_run: ${{ steps.parse.outputs.should_run }}
+    steps:
+      - name: Parse comment
+        id: parse
+        run: |
```

actionlint (1.7.7)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)